src/scripts/main.js

```diff
@@ -1,3 +1,304 @@
 'use strict';
 
-// write code here
+const body = document.body;
+
+const tableHeader = document.querySelector('thead');
+
+const tableBody = document.querySelector('tbody');
+
+const tableBodyElements = tableBody.getElementsByTagName('tr');
```

Comment:
```suggestion
const rows = tableBody.getElementsByTagName('tr');
```
Краще давати назви, які дійсно означають те, що містять